### ISSUE TITLE
Test Edit Profile Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4914]
# Page to Test:

Edit Profile - https://openlibrary.org/people/bpmcneilly?m=edit

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by bpmcneilly** [comment]
@Sabreen-Parveen all yours!

**Comment by shaiksahilrizwan** [comment]
@RayBB Hi! I'd like to claim this Good First Issue. I will test the Edit Profile page for missing aria-labels, missing form labels, and focus states, and open a PR with the accessibility fixes!

**Comment by shaiksahilrizwan** [comment]
@RayBB Hi, I’d like to work on this issue.
I’ve started testing the Edit Profile page for accessibility and will report any issues I find as separate tickets as per the instructions.

**Comment by shaiksahilrizwan** [comment]
I have found an accessibility violation regarding disconnected form labels on the Edit Profile page and opened a separate issue for it here: [#12554](https://github.com/internetarchive/openlibrary/issues/12554)

**Comment by shaiksahilrizwan** [comment]
@RayBB I would love to work on the fix for this! Could I be assigned to #12554 so I can open a PR for it?

**Comment by RayBB** [comment]
https://github.com/internetarchive/openlibrary/issues/12554 is closed so this can be closed too.
